### ISSUE TITLE
Very High CPU Usage FTB

### ISSUE THREAD

**Issue by jloleysens** [itzg/dockerfiles#333]
Hi,

I am trying to run mc server image with an [FTB modpack](https://www.feed-the-beast.com/projects/ftb-beyond/files/2482845) but have a situation where CPU usage skyrockets for ~5 minutes and then the container dies before the mc server starts (see attached log for more information).

I am running the following cmd:

```
docker run -d -v /var/minecraft:/data  \
    -e TYPE=FTB -e FTB_SERVER_MOD=https://www.feed-the-beast.com/projects/ftb-beyond/files/2482845 \
    -p 25565:25565 -e EULA=TRUE --name mc itzg/minecraft-server
```

Without the modpack added as above the server runs fine (i.e., in vanilla). I also tried using the `ftb` tagged image without a different result. So I am not sure whether this may be something wrong with the pack?

Update:

Using `docker stats` I was able to see that the container was running with very high/maxed out CPU usage.

Any guidance would be great, thanks!

[log.txt](https://github.com/itzg/dockerfiles/files/3269880/log.txt)

**Comment by itzg** [comment]
A lot of the FTB modpacks seems to be a pretty intense combination of mods -- they're cool, but I tend to avoid them due to the bulk. Unfortunately that means I don't have much specific advice other than suggesting trying to increase the allowed heap size if you have RAM available by setting the `MEMORY` environment variable something more than the default of "1G". I know that's a long shot since it seems to be CPU bound, but might happen to help.

**Comment by jloleysens** [comment]
@itzg Thanks for the advice.

I did up the RAM but it still got stuck - when I ran the same FTB server on a VM setup outside of docker I was able to get it working. I must also add I am new to running modded MC servers and definitely under estimated that added resource usage 😄 .

For now I am going to run vanilla (due to other reasons). Might get back to running the modded set up and if I do so I'll update my findings here. Closing for now.
